esx_service/utils/auth.py

```diff
@@ -326,6 +326,25 @@ def add_volume_to_volumes_table(tenant_uuid, datastore, vol_name, vol_size_in_MB
 
     return None
 
+def remove_volume_from_volumes_table(tenant_uuid, datastore, vol_name):
```

Comment:
I don't think it's a good idea to include the database table name in the method name. DB schema should be isolated and decoupled from core business logic so that both can evolve independently. So I'd suggest to rename this method to "remove_volume"